<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>661</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="dealerLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>661</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="playerLayout"/>
   </widget>
   <widget class="QPushButton" name="increaseBetBtn">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>414</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Increase bet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decreaseBetBtn">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>494</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrease bet</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>343</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="hitButton">
       <property name="text">
        <string>Hit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="standButton">
       <property name="text">
        <string>Stand</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="dealerScoreLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>337</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dealer score: </string>
    </property>
   </widget>
   <widget class="QLabel" name="ballanceLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>283</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your ballance: </string>
    </property>
   </widget>
   <widget class="QLabel" name="betLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>256</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bet: </string>
    </property>
   </widget>
   <widget class="QLabel" name="playerScoreLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your score: </string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>50</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="deckLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
